distributed/rpc/pipeline/main.py

```diff
@@ -219,7 +219,7 @@ def run_master(split_size):
 def run_worker(rank, world_size, num_split):
     os.environ['MASTER_ADDR'] = 'localhost'
     os.environ['MASTER_PORT'] = '29500'
-    options = rpc.TensorPipeRpcBackendOptions(num_worker_threads=256)
+    options = rpc.TensorPipeRpcBackendOptions(num_worker_threads=256, rpc_timeout=300)
```

Comment:
Could you add a comment here explaining why we need the 300s timeout?

Reply:
Updated the commit with comments to explain the change.